css/index.css

```
@@ -18,6 +18,8 @@ header{
           
 }
 
+/* Navigaition-------------------------------------------------------------------*/
```

There's a typo here.

```suggestion
/* Navigation-------------------------------------------------------------------*/
```